data/resources/ui/content-message-media.ui

```diff
@@ -1,49 +1,8 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <interface>
-  <template class="ContentMessageMedia" parent="GtkWidget">
-    <style>
-      <class name="message-bubble"/>
-      <class name="message-media"/>
-    </style>
-    <child>
-      <object class="GtkBox" id="content">
-        <property name="orientation">vertical</property>
-        <child>
-          <object class="GtkOverlay">
-            <child type="overlay">
-              <object class="GtkProgressBar" id="progress_bar">
-                <property name="halign">center</property>
-                <property name="valign">center</property>
-              </object>
-            </child>
-            <child>
-              <object class="ContentMediaPicture" id="picture">
-                <property name="overflow">hidden</property>
-                <style>
-                  <class name="media-picture"/>
-                </style>
-              </object>
-            </child>
-          </object>
-        </child>
-        <child>
-          <object class="GtkLabel" id="caption_label">
-            <property name="selectable">True</property>
-            <property name="use-markup">True</property>
-            <property name="wrap">True</property>
-            <property name="wrap-mode">word-char</property>
-            <property name="xalign">0</property>
-            <property name="margin-top">6</property>
-            <property name="margin-bottom">4</property>
-            <property name="margin-start">10</property>
-            <property name="margin-end">10</property>
-            <property name="visible">False</property>
-            <style>
-              <class name="message-text"/>
-            </style>
-          </object>
-        </child>
-      </object>
-    </child>
+  <template class="ContentMessageMedia" parent="ContentMessageRow">
+    <property name="content">
+      <object class="ContentMessageMediaContent" id="content"/>
+    </property>
```

Comment:
What was your intention in not unifying this file with `content-media-message-content.ui`?

Reply:
The `ContentMessageMediaContent` needs a custom sizing implementation, so I had to create a separate custom widget just for that reason.

Reply:
Oh yes, I think you told that once. I hope I don't ask that again in a week. :-p